app/controllers/api_users_controller.rb

```diff
@@ -28,7 +28,10 @@ def create
 private
 
   def post_to_endpoint(user)
-    @user = { email: user.email, department: user.department, service: user.service }
+    @user = { email: user.email,
+              department: user.department,
+              service: user.service,
+              is_government: user.is_government == 'yes' }
```

Comment:
What is the intention here?

Reply:
we are marshalling the form response `yes` or `no` into a boolean.